src/main/java/com/patterns/prototype/SheetOfPaper.java

```diff
@@ -0,0 +1,20 @@
+package com.patterns.prototype;
+
+import lombok.ToString;
+
+@ToString
+public class SheetOfPaper implements Copyable {
```

Comment:
Why not use standard Clonable interface?

Reply:
I read that I can use my own interface. If it is not, I can rewrite the code?

Reply:
You could by why not to use a standard way?

Reply:
We used the Clonable interface in the previous homework when we created immutable class , so I decided to use another interface, create my own. I saw an example on the Internet when I looked at the information for a lesson.